.github/workflows/test-e2e-pr.yml

```
@@ -4,106 +4,215 @@ on:
+      - name: Find changed components
+        id: find-changed-components
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const changedFiles = '${{ steps.changed-files.outputs.all_changed_files }}'.split(',').filter(file => file.trim() !== '');
+            console.log('变更的文件列表:', changedFiles);
+
+            // 增强组件路径匹配规则，确保能正确识别各种路径格式
+            const componentPathRules = [
+              // 匹配 packages/renderless/src/{组件名}/ 路径
+              { 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)